client/src/components/Sharing/EditableUrl.vue

```diff
@@ -84,6 +84,7 @@ function onCopyOut() {
         </BButton>
 
         <BButton
+            v-if="!editing"
```

Comment:
Maybe it'd be better to just disable the button ?

Reply:
I think it better to hide things that are not relevant to the current view. And when editing slug copying of the full url is not relevant.

Reply:
In this case, there is not too much of a difference, but there are clear pros and cons for both options so we shouldn't always prioritize one or the other, but rather choose on a case-by-case basis.
Here is an article without the aim to bikeshed too much :stuck_out_tongue_winking_eye:  https://uxpsychology.substack.com/p/hidden-vs-disabled-states

Reply:
I see both sides and my preference was in the commit. However since there are two votes for the other option I added cd7de5511d2ff757fee04f8c982153c4a0050930